### ISSUE TITLE
Add gradient color to tabIndicator

### ISSUE THREAD

**Issue by chiragmittal19** [madhu314/react-native-tabbed-view-pager-android#12]
Is there any way to put linear gradient as tabIndicatorColor?

**Comment by ashokkumar88** [comment]
Yes. Just create a View with linear gradient and apply style `position:"absolute,height:56"` to the View. Then below this View place `TabbedViewPager`

**Comment by chiragmittal19** [comment]
@ashokkumar88 But that will cover the complete tab bar. I want to customize only the line under selected tab.

**Comment by ashokkumar88** [comment]
@chiragmittal19 Then disable the default `tabIndicator` and put your custom indicator in a View and animate with `onPageScroll` event.

**Comment by madhu314** [comment]
This feature is not planned for this library.
